src/app/components/challenge-create/challenge-create.component.html

```diff
@@ -1,27 +1,56 @@
-<app-header-static>
-</app-header-static>
-<div class="container-top">
-  <div class="create-challenge-container">
-    <div class="create-challenge-title">
-    	Create Challenge
-    </div>
-     <app-forcelogin [path]="'/challenge-create'" *ngIf="!authServicePublic.isLoggedIn()"></app-forcelogin>
-    <div class="card card-light" *ngIf="authServicePublic.isLoggedIn()">
-    	
-    <div class="create-form">
-        <div class="title">Please upload zip file to create a challenge!</div>
-    	<app-input [isRequired]="true" [label]="'zip_configuration'" [placeholder]="'Upload File *'" [type]="'file'" [theme]="'dark'"  #formcreate></app-input>
-    	<div class="submit-btn-div">
-          <span class="btn btn-filter selected" (click)="formValidate('formcreate')">Submit</span>
+<app-side-bar *ngIf="authService.isAuth"></app-side-bar>
+<app-header-static></app-header-static>
+<div class="web-container" [style.center]="!authService.isAuth">
+  <div class="challenge-create-flex">
+    <div class="challenge-create-content">
+
+      <section class="ev-md-container text-center ">
+        <div class="row">
+          <div class="col s12 m9 offset-m3">
+            <div class="zip-file-title">Please upload zip file to create a challenge!</div>
+            <div class="file-field input-field col s6">
+              <form name="ChallengeCreateForm" #ChallengeCreateForm="ngForm" (ngSubmit)="challengeCreate()" novalidate>
+                <div class="waves-effect waves-dark btn ev-btn-dark w-300 fs-14">
+                  <span>Upload File</span>
+                  <input name="input_file" (change)="handleUpload($event)" accept=".json, .zip, .txt" type="file" class="text-dark-black dark-autofill w-400" [(ngModel)]="ChallengeCreateForm['input_file']">
+                </div>
+
+                <div class="file-path-wrapper">
+                  <input disabled class="file-path validate" name="file_path" type="text">
+                </div>
+
+                <div class="align-left reg-control">
+                  <button class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14" type="submit" value="Submit">Submit</button>
+                </div>
+
+                <div *ngIf="isFormError" class="wrn-msg text-highlight align-right"> Please Upload File </div>
```

Comment:
The position of this is not aligned correctly. Render it directly below the `Upload file` text field.

Reply:
Done